manifest.json

```diff
@@ -0,0 +1 @@
+{"key":"value"}
```

Comment:
This shouldn't be here, sorry for that. I should write the manifest in the dest folder so that it gets deleted.